src/app/components/layout/header/header.component.html

```diff
@@ -41,81 +41,159 @@
   <div class="moj-primary-navigation__container">
     <div class="moj-primary-navigation__nav">
       <nav class="moj-primary-navigation" aria-label="Primary navigation">
-        <ul class="moj-primary-navigation__list">
-          <li class="moj-primary-navigation__item">
-            <a
-              class="moj-primary-navigation__link"
-              routerLink="search"
-              routerLinkActive="active"
-              ariaCurrentWhenActive="page"
-              >Search</a
-            >
-          </li>
-          @if (userService.isTranscriber()) {
+        @if (userService.isAdmin() && router.url.startsWith('/admin')) {
```

Comment:
Is it worth splitting this out into `portal-header` and `admin-header` components?

Reply:
100% Yes - I think it should be done in another PR / ticket though. I tried to keep this PR from getting too big and get it in asap so we can get cracking with the rest of the admin screens.